qir/qsharp/ConstArray/validation.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
I am not sure what does file does/ is used for?

Reply:
This is output from testing whether or not the output IR is compliant with a given profile. This file should not be checked in and has been removed.